src/contribution.c

```diff
@@ -0,0 +1,35 @@
+/******************************************************
+* Author : Amuleen Gulati 
```

Comment:
This won't compile in Doxygen.

Reply:
Thanks! I just realized that and fixed it.